addons/sourcemod/translations/zombieriot.phrases.txt

```diff
@@ -493,28 +543,32 @@
 		"pl"		"Ktoś użył twojego pudła z amunicją, otrzymujesz +20 Kredytów!"
 		"es"		"Alguién usó tu caja de munición, ganas 20 dolares"
 		"fr"		"Quelqu'un a utilisé votre boite de munitions, vous gagnez 20 dollars!"
+		"pt"		"Alguém usou sua caixa de munição, você ganhou +20 Créditos"
 	}
 	"Armor Table Used"
 	{
 		"en"		"Someone used your armor table, you gain +40 Credits!"
-	//	"it"		"Qualcuno ha usato la tua cassa di munizioni, hai ricevuto +40 Crediti!"
+		"it"		"Qualcuno ha usato la tua cassa di munizioni, hai ricevuto +40 Crediti!"
 		"ru"		"Кто-то воспользовался вашим верстаком брони, вы получили +40 кредитов!"
-	//	"pl"		"Ktoś użył twojego pudła z amunicją, otrzymujesz +40 Kredytów!"
-	//	"es"		"Alguién usó tu caja de munición, ganas 40 dolares"
```

Comment:
also needs to be recommented

Reply:
solved in https://github.com/StShAzzy/TF2-Zombie-Riot/commit/a896378e6b917847fcf633b1a188fd9a06158425